### PR TITLE
OK-30182, OK-30210: Algorand walletconnect & Increase maximum fee rate for neurai

### PR DESCRIPTION
**Pull request by originalix**

### CODE REVIEW

**Review by sidmorizon — DISMISSED** [review]

File: packages/shared/src/utils/uriUtils.ts [hunk 1]
```diff
@@ -222,6 +223,24 @@ function buildDeepLinkUrl<T extends EOneKeyDeepLinkPath>({
...
+};
+function safeGetWalletConnectOrigin(proposal: Web3WalletTypes.SessionProposal) {
+  try {
+    const { origin } = new URL(proposal.params.proposer.metadata.url);
+    return origin;
+  } catch (err) {
+    const nameToUrl =
+      NameToUrlMapForInvalidDapp[proposal.params.proposer.metadata.name];
+    if (nameToUrl) {
+      const { origin } = new URL(nameToUrl);
```
Comment (sidmorizon) [inline packages/shared/src/utils/uriUtils.ts]: try catch

File: packages/shared/src/utils/uriUtils.ts [hunk 2]
```diff
@@ -222,6 +223,24 @@ function buildDeepLinkUrl<T extends EOneKeyDeepLinkPath>({
...
+const NameToUrlMapForInvalidDapp: Record<string, string> = {
+  'Algorand Governance': 'https://governance.algorand.foundation',
+};
+function safeGetWalletConnectOrigin(proposal: Web3WalletTypes.SessionProposal) {
+  try {
+    const { origin } = new URL(proposal.params.proposer.metadata.url);
+    return origin;
+  } catch (err) {
+    const nameToUrl =
+      NameToUrlMapForInvalidDapp[proposal.params.proposer.metadata.name];
```
Comment (sidmorizon) [inline packages/shared/src/utils/uriUtils.ts]: `${name}--${description}` 加上 description 判断

File: packages/shared/src/utils/uriUtils.ts (hunk 1 above)
Comment (originalix) [reply packages/shared/src/utils/uriUtils.ts]: Fixed

File: packages/shared/src/utils/uriUtils.ts (hunk 2 above)
Comment (originalix) [reply packages/shared/src/utils/uriUtils.ts]: Fixed

**Review by sidmorizon — APPROVED** [review]

**Review by revan-zhang — APPROVED** [review]

**Pull request merged**